client/extensions/woocommerce/app/order/create.js

```diff
@@ -34,18 +39,37 @@ import { recordTrack } from 'woocommerce/lib/analytics';
 import { sendOrderInvoice } from 'woocommerce/state/sites/orders/send-invoice/actions';
 
 class Order extends Component {
+	possiblyFetchDefaultCurrency( props ) {
+		const { currencyCode, orderEdits, settingsGeneralLoaded, siteId } = props;
+		if ( siteId ) {
+			if ( ! settingsGeneralLoaded ) {
+				this.props.fetchSettingsGeneral( siteId );
+			} else if ( isEmpty( orderEdits.currency ) ) {
+				this.props.editOrder( siteId, { currency: currencyCode } );
+			}
```

Comment:
I think I've figured out the logic here but maybe add a comment or two with some plain english for what this condition is doing?

Reply:
Good idea! Fixed in https://github.com/Automattic/wp-calypso/pull/23568/commits/8a7c403776596e9294f02baf3bf29b7b5c04d140

Reply:
Added comments at top of this function